module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,8 @@
+#include "vowel.hpp"
+
+void removeVowels(std::vector<std::string>& strings){
+    constexpr std::array<char,vowels_number> vowels ={'a','e','i','o','u','y','A','E','I','O','U','Y'};
```

Comment:
I would consider moving this to the header.

Reply:
You need it only in this one function. I will keep it here.